backend/api/tests/test_models.py

```
@@ -0,0 +1,222 @@
+            email="company2@example.com",
+            firebase_uid="companyuid2",
+            role="company"
+        )
+        profile2 = CompanyProfile.objects.create(
+            user=user2
+        )
+        assert str(profile2) == f"Profile for company2@example.com"
+
+@pytest.mark.django_db
+class TestViewerProfileModel:
+    def test_viewer_profile_creation(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_viewer_profile_creation`

Add return type annotation: `None`

(ANN201)